### PR TITLE
add localhost to django setting's allowed_host

### PR DESCRIPTION
**Pull request by ibrahimyaacob92**

### CODE REVIEW

**Comment by rabbilyasar** [comment]
Task linked: [CU-95utv9 Add 'localhost' to allowed host](https://app.clickup.com/t/95utv9)

**Review by rabbilyasar — CHANGES_REQUESTED** [review]
add a space after comma.

File: backend/backend/settings.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['0.0.0.0']
+ALLOWED_HOSTS = ['0.0.0.0','localhost']
```
Comment (ibrahimyaacob92) [inline backend/backend/settings.py]: @rabbilyasar you mean here ?

Comment (rabbilyasar) [reply backend/backend/settings.py]: Yes between the host addresses add a space after the comma, that's a convention while writing lists. We will be following as much conventions as possible.

**Pull request closed without merging**